mm2src/coins/utxo/slp.rs

```diff
@@ -2022,7 +2016,7 @@ mod slp_tests {
 
         let err = fusd.send_raw_tx(&tx_bytes_str).wait().unwrap_err();
         println!("{:?}", err);
-        assert!(err.contains("is not valid with reason outputs greater than inputs"));
+        assert!(err.contains("the transaction was rejected by network rules.\\n\\ntransaction already in block chain"));
```

Comment:
This had to change since `send_raw_tx` now calling `uxto_common::send_raw_tx` [here](https://github.com/KomodoPlatform/atomicDEX-API/pull/1245/files#diff-f7bb233ee3a45b82214eecd819ec1c748eec916a6037c5fc98c665e1f52c50e7R1086) @artemii235

Reply:
@ozkanonur Very good that we have this test 🙂 
I forgot that SLP requires additional transaction validation and provided incorrect comment that `utxo_common::send_raw_tx_bytes` can be used for it. Due to the missing validation, the invalid transaction was broadcast - with SLP protocol, it effectively means tokens burning (luckily, test net tokens in this case). More info on the topic: https://slp.dev/guides/slp-implementation-instructions/#have-multiple-sources-of-validation-or-client-side-validation

The usage of validation code
https://github.com/KomodoPlatform/atomicDEX-API/blob/ff6fc9f6722cf4f8ac0a290043deded201e7dc59/mm2src/coins/utxo/slp.rs#L1037-L1040

Could you please roll back the `send_raw_tx` implementation and use `broadcast_tx` in the `Slp::send_raw_tx_bytes` too? Please also extend this test and check that `send_raw_tx_bytes` will return `is not valid with reason outputs greater than inputs` for this tx.

Also, if the test started to fail, it's undesirable to change it - there's a very high chance that tested code got broken.

Reply:
Done